frontends/web/src/api/account.ts

```diff
@@ -317,4 +317,8 @@ export const addAccount = (coinCode: string, name: string): Promise<TAddAccount>
     coinCode,
     name,
   });
+};
+
+export const testRegister = (pin: string): Promise<null> => {
```

Comment:
nit: this belongs into the backend.ts not account.ts. account.ts is for account endpoints, which all have a account `code` param. same for `addAccount` above actually.